extract/extract-handlers.ts

```diff
@@ -54,10 +54,7 @@ export function extractHandlers(path: string) {
 						const fullDefinition: FullHandlerDefinition<ApiHandlerDefinition> =
 							{
 								...definition,
-								name: `${definition.method}${definition.route.replace(
-									/\//g,
-									'-',
-								)}`,
+								name: pascalCase(`Api ${definition.name}`),
```

Comment:
@loganhuskins  This was changed since last review. I'm now forcing the names to be pascal cased and prefixed with "Api"